api/tests/services/rag/test_query_rewriter.py

```
@@ -0,0 +1,230 @@
+        result = await rewriter.rewrite(
+            "What about the security deposit stuff?",
+            chat_history=history,
+        )
+        # Should have attempted LLM (mocked to return rewritten)
+        assert result.rewritten is True
+        assert result.strategy in ("llm", "heuristic")
+
+    @pytest.mark.asyncio
+    async def test_llm_timeout_falls_back(self, rewriter_settings):
+        slow_client = MagicMock()
+        slow_client.chat.completions.create.side_effect = lambda **kwargs: (
```

Ruff (0.15.0)

[warning] Unused lambda argument: `kwargs`

(ARG005)